### ISSUE TITLE
Map is frozen after an unexpeted error

### ISSUE THREAD

**Issue by remiOdite** [thedirtyfew/dash-leaflet#96]
Hi,

we encounter an issue we do not succeed to solve. After a while, the dash leaflet map is like frozen. We cannot moove the map using the mouse drag. We can only zoom in/out using the scroll.

It seems the map is frozen after this error in js console :
```
Uncaught TypeError: a is undefined
    getClusters https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:86
    value https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:86
    fire https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onDragEnd https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    fire https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    finishDrag https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onUp https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    a https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ut https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    r https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Nt https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Nt https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ce https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Oe https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onDown https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    a https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ut https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    i https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
dash_leaflet.v0_1_15m1625043721.min.js:86:341464
    getClusters https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:86
    value https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:86
    value self-hosted:1178
    fire https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onDragEnd https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    fire https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    finishDrag https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onUp https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    a https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ut https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    r https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    (Asynchrone : EventListener.handleEvent)
    Nt https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Nt https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ce https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Oe https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    _onDown https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    a https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    Ut https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    i https://remidash.intra.odit-e.com/_dash-component-suites/dash_leaflet/dash_leaflet.v0_1_15m1625043721.min.js:6
    <anonyme> self-hosted:1178
```

Is anyone faced this issue ? Do you have any idea how to solve it ? Maybe the issue is not linked to the above error.

Thanks in advance,
regards

**Comment by emilhe** [comment]
Could you post an MWE?

**Comment by remiOdite** [comment]
Ok, I succeed to reproduce the bug, and understand exactly when it happens.

There is a simple script if you want to test :
```
#!/usr/bin/env python3
import dash_html_components as html
import dash_leaflet as dl
import dash_leaflet.express as dlx
from dash import Dash

app = Dash()
app.layout = html.Div([
    dl.Map([
        dl.TileLayer(),
        # From in-memory geojson. All markers at same point forces spiderfy at any zoom level.
        dl.GeoJSON(data=dlx.dicts_to_geojson([dict(lat=-37.8+i/1000, lon=175.6+i/1000) for i in range(0,20)]),
                        zoomToBoundsOnClick=True, 
                   cluster=True),
    ], center=(-37.75, 175.4), zoom=11, style={'width': '100%', 'height': '50vh', 'margin': "auto", "display": "block"}),
])

if __name__ == '__main__':
    app.run_server()
```

Run the app, then, click on one cluster in order to trigger the ZoomToBoundsOnClick animation. While the animation occurs, perform a drag with the mouse to move the map elsewhere before the animation ends.

The error will appear, and then map will be 'frozen' after that.

I am using dash-leaflet==0.1.15

**Comment by emilhe** [comment]
The bug should be fixed in `dash-leaflet==0.1.16`.

**Comment by remiOdite** [comment]
Thank you !
